### DISCUSSION TITLE
Export as PDF

### DISCUSSION THREAD

**Discussion by Lanhild** [growilabs/growi#8789] [Ideas / Requests] [answered]
We currently can export a page as Markdown. It'd be very useful to be able to export it as PDF.

**Comment by yuki-takei** [comment]
It is Work In Progress under https://github.com/weseek/growi/tree/feat/page-bulk-export

However, since the implementer is not a full-time committer, the completion date has not yet been determined.

**Comment by yuki-takei** [comment, marked as answer]
`growilabs/pdf-converter` has been released with [v7.2.1](https://github.com/growilabs/growi/releases/tag/v7.2.1)
